datumaro/plugins/data_formats/datumaro_binary/format.py

```diff
@@ -23,6 +23,7 @@ class DatumaroBinaryPath:
     SECRET_KEY_FILE = "secret_key.txt"
 
     MAX_BLOB_SIZE = 2**20  # 1 Mega bytes
+    MP_TIMEOUT = 30.0  # 30 secs
```

Comment:
Is there any reason to consider `MP_TIMEOUT` as 30 secs?

Reply:
No reason, I just put in a value that I feel moderately long.